src/amo/components/RatingsByStar/index.js

```diff
@@ -108,17 +116,19 @@ export class RatingsByStarBase extends React.Component<InternalProps> {
             let starCount;
             let starCountNode;
 
-            function createLink(text) {
+            function createLink(text, count, tab = true) {
               invariant(addon, 'addon was unexpectedly empty');
 
               return (
                 <Link
-                  title={linkTitles[star] || ''}
+                  title={getLinkTitle(star, count) || ''}
                   to={reviewListURL({
                     addonSlug: addon.slug,
                     score: star,
                     src: location.query.src,
                   })}
+                  aria-hidden={!tab}
```

Comment:
A boolean attribute should only be present when true. 

> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

See https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes for more details.

Reply:
Ohk I will update this change. Good to know this as I don't have knowledge about this. Thanks :+1: